custom_components/securitas_direct/manifest.json

```diff
@@ -1,12 +1,10 @@
 {
   "domain": "securitas_direct",
   "name": "Securitas Direct",
-  "documentation": "https://github.com/segalion/securitasdirect",
-  "issue_tracker": "https://github.com/segalion/securitasdirect/issues",
+  "documentation": "https://www.home-assistant.io/integrations/securitas_direct",
```

Comment:
You are replacing the original documentation with a broken link, I think you need to undo this change (and issue_tracker as well).

Reply:
I'm following HA guidelines (https://developers.home-assistant.io/docs/creating_integration_manifest) in order to submit the integration.

Reply:
Ok. Sorry, I didn't see the issue #23 
BTW, I noticed that after changing to your code, now the HA component appears as "unnamed device" and has no unique id, do you know how to fix it?

Reply:
It's probably due the fact I've removed this
`'alias': hub.session.alias` from `device_state_attributes`

Alias comes from an operation ("INS"). I'll try to add that at the beginning of the connection and merge it here asaic.

Reply:
Yes! This last commit did fix it. Awesome. Thanks!